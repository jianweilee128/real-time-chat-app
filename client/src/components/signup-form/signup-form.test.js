import React from "react";
import { shallow } from "enzyme";

import SignupForm from "./signup-form.component";

describe("SignupForm component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<SignupForm.WrappedComponent />);
  });

  it("should render SignupForm component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
