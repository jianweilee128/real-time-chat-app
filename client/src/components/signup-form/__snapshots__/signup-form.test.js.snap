// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupForm component should render SignupForm component 1`] = `ShallowWrapper {}`;
